src/cleanfigure.m

```diff
@@ -50,15 +50,15 @@ function cleanfigure(varargin)
 
   % Set up command line options.
   m2t.cmdOpts = m2tInputParser;
-  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'minimumPointsDistance', 1.0e-10, @isnumeric);
   m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'handle', gcf, @ishandle);
+  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'targetResolution', '15x9@600',@isstr);
```

Comment:
It is advised to use `@ischar` for checking a string. 

Anyhow, I think we're better off with a more structured format that we can validate more easily. I'd propose something like `[15 9 600]` or `{15, 9, 600}`. Those we can check, and we don't need to parse them at the time we use them.

We also have to provide some backward compatibility: the `minimumPointsDistance` parameter should remain valid for at least 1 release (even if it doesn't do anything). So I would propose to add:

```
m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'minimumPointsDistance', 1.0e-10, @isnumeric);

m2t.cmdOpts = m2t.cmdOpts.deprecateParam(m2t.cmdOpts, 'minimumPointsDistance', 'targetResolution')
```